examples/construct_lmdb_load_scaler.py

```diff
@@ -0,0 +1,128 @@
+        subdir=False,
+        meminit=False,
+        map_async=True,
+    )
+
+    # Define symmetry functions
+    Gs = {
+        "default": {
+            "G2": {
+                "etas": np.logspace(np.log10(0.05), np.log10(5.0), num=8),
+                "rs_s": [0, 0.1, 1.0],
+            },
+            "G4": {"etas": [0.005], "zetas": [1.0, 4.0], "gammas": [1.0, -1.0]},
+            "cutoff": 6,
+        },
+    }
+
+    training_atoms = images
+    elements = np.array([atom.symbol for atoms in training_atoms for atom in atoms])
+    elements = np.unique(elements)
+    descriptor = Gaussian(Gs=Gs, elements=elements, cutoff_func="Cosine")
+    descriptor_setup = ("gaussian", Gs, {"cutoff_func": "Cosine"}, elements)
+    forcetraining = True
+
+    a2d = AtomsToData(
+        descriptor=descriptor,
+        r_energy=True,
+        r_forces=True,
+        save_fps=False,
+        fprimes=forcetraining,
+    )
+
+    data_list = []
+    idx = 0
+    for image in tqdm(
+        images,
+        desc="calculating fps",
+        total=len(images),
+        unit=" images",
+    ):
+        do = a2d.convert(image, idx=idx)
+        data_list.append(do)
+        idx += 1
+
+    normalizers = torch.load(scaler_path)
+    feature_scaler = normalizers["feature"]
+    target_scaler = normalizers["target"]
+
+    feature_scaler.norm(data_list)
+    target_scaler.norm(data_list)
+
+    idx = 0
+    for do in tqdm(data_list, desc="Writing images to LMDB"):
+        txn = db.begin(write=True)
+        txn.put(f"{idx}".encode("ascii"), pickle.dumps(do, protocol=-1))
+        txn.commit()
+        idx += 1
+
+    txn = db.begin(write=True)
+    txn.put("feature_scaler".encode("ascii"), pickle.dumps(feature_scaler, protocol=-1))
+    txn.commit()
+
+    txn = db.begin(write=True)
+    txn.put("target_scaler".encode("ascii"), pickle.dumps(target_scaler, protocol=-1))
+    txn.commit()
+
+    txn = db.begin(write=True)
+    txn.put("length".encode("ascii"), pickle.dumps(idx, protocol=-1))
+    txn.commit()
+
+    txn = db.begin(write=True)
+    txn.put("elements".encode("ascii"), pickle.dumps(elements, protocol=-1))
+    txn.commit()
+
+    txn = db.begin(write=True)
+    txn.put(
+        "descriptor_setup".encode("ascii"), pickle.dumps(descriptor_setup, protocol=-1)
+    )
+    txn.commit()
+
+    db.sync()
+    db.close()
+
+
+if __name__ == "__main__":
+    scaler_path = "./normalizers.pt"
+    images = []
+
+    distances = np.linspace(2, 5, 10000)
+    for dist in distances:
+        image = Atoms(
+            "CuCO",
+            [
+                (-dist * np.sin(0.65), dist * np.cos(0.65), 0),
+                (0, 0, 0),
+                (dist * np.sin(0.65), dist * np.cos(0.65), 0),
+            ],
+        )
+        image.set_cell([10, 10, 10])
+        image.wrap(pbc=True)
+        image.set_calculator(EMT())
+        image.get_potential_energy()
+        images.append(image)
+
+    construct_lmdb(images, scaler_path, lmdb_path="./data2.lmdb")
```

Comment:
Is this only difference between this and the other `construct_lmdb.py` is that this one assumes a normalizer is saved to disk and just loads that instead? If so, can we consolidate the 2 codes and just add an argument instead to do normalization or not.

Reply:
yes, indeed. 
I kept them separate to make it more clear, cuz presumably the users need to run them separately. 
we can consolidate them tho if that makes more sense

Reply:
consolidated